setup.py

```diff
@@ -64,8 +64,8 @@ def run_tests(self):
     raise Exception('this library does not support python >2 and <3.5')
```

Comment:
I don't follow.

The old error message implied Python 2.7 wasn't supported, but it is.

Both of the proposed error messages don't comment about Python 2.x, but do say that 3.0-3.4 are not supported, which seems to have been the intention all along.